waku/waku_discv5.nim

```diff
@@ -314,8 +320,12 @@ proc subscriptionsListener*(wd: WakuDiscoveryV5, topicSubscriptionQueue: AsyncEv
     if unsubRes.isErr():
       debug "ENR shard removal failed", reason= $unsubRes.error
 
-    if subRes.isOk() and unsubRes.isOk():
-      debug "ENR updated successfully"
+    if subRes.isErr() and unsubRes.isErr():
```

Comment:
Should this better be an `or` ?

Reply:
if either one succeed then add the log message and update the predicate.

Reply:
Failing is not that bad in this case.